lib/code_corps/stripe_testing/charge.ex

```diff
@@ -0,0 +1,7 @@
+defmodule CodeCorps.StripeTesting.Charge do
+  import CodeCorps.StripeTesting.Helpers
+
+  def retrieve(_id, _opts) do
+    {:ok, load_fixture(Stripe.Charge, "charge")}
+  end
```

Comment:
```Elixir
def retrieve(id, _opts) do
  {:ok, load_fixture(Stripe.Charge, "charge")
end
```

Would allow us to, in the future, specify ID and let stripe "retrieve" a specific fixture. We don't need it right now, but at the same time, the only fixture we try to retrieve right now is "charge".